app/controllers/ReportOneLoginProblemController.scala

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import config.AppConfig
+import connectors.deskpro.DeskproTicketQueueConnector
+import model.{ContactPreference, ContactPreferenceFormatter, DateOfBirth, EmailPreference, ReportOneLoginProblemForm}
+import play.api.data.Form
+import play.api.data.Forms.*
+import play.api.i18n.I18nSupport
+import play.api.mvc.*
+import services.DeskproSubmission
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import util.{DateOfBirthValidator, DeskproEmailValidator, NameValidator, TaxIdentifierValidator}
+import views.html.{InternalErrorPage, ReportOneLoginProblemConfirmationPage, ReportOneLoginProblemPage}
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+object ReportOneLoginProblemFormBind {
```

Comment:
I think we could define the form in the controller rather than having a separate object, or have it in a companion object of the controller - but since the controller is singleton it seems ok to define it in there